templates/grid_broker/grid_broker.py

```diff
@@ -347,19 +377,30 @@ def _is_locked(self, tx):
 <html>
 
 <body>
-    <h1>We could not complete your reservation at this time</h1>
+    <h1>We could not {action} your reservation at this time</h1>
     <div class="content">
-        <p>Unfortunately, we could not complete your reservation. We will refund your reservation to {address}. Please try again at a later time</p>
+        <p>Unfortunately, we could not {action} your reservation. We will refund your reservation to {address}. Please try again at a later time</p>
     </div>
     <div class="error">
         <h3>Error detail:</h3>
         <ul>
             <li>
-                <p>transaction ID of the reservation: <em>{tx_id}</em></p>
+                <p>transaction ID of the {type}: <em>{tx_id}</em></p>
             </li>
             <li>error: <code>{error}</code></li>
         </ul>
     </div>
 </body>
 </html>
 """
+
+_extend_template = """
+<html>
+<body>
+    <h1>Your reservation {tx_id} has been extending successfully</h1>
```

Comment:
can we also specify what kind of reservation has been extended ?
Only the transaction ID is not really explicit.